2-Browsers/Week1/test-reports/ex1-bookList.fail.txt

```diff
@@ -0,0 +1,3 @@
+*** Unit Test Error Report ***
+
+- Generated HTML should include an <li> with title and author for each book
```

Comment:
The software library used by the unit tests to inspect the generated HTML does not support `innerText` . If you replace `innerText` with `textContent` then this "error" will no longer be reported. I will add an ESLint rule to warn about this so that the next class will not run into this. (In the past, `innerText` was not non-standard, whereas `textContent` was.)